### ISSUE TITLE
"focused" property not changing in wayland

### ISSUE THREAD

**Issue by maru-sama** [mpv-player/mpv#11694]
### Important Information

Provide following Information:
- Latest git
- Ubuntu 
- build from source
- Gnome

If you're not using git master or the latest release, update.
Releases are listed here: https://github.com/mpv-player/mpv/releases

### Reproduction steps

Start mpv and then move the focus to another window
### Expected behavior
The "focused" property should change to false
### Actual behavior
The focused property does not change at all

If I try the same steps under X the property change triggers correctly.

**Comment by Dudemanguy** [comment]
Indeed, it doesn't work in mutter. It works fine in sway and weston though so I'm not sure what would be different here. Note that wayland doesn't actually support querying if a window is focused, we're currently guessing.
